docs/pages/eas/hosting/reference/worker-runtime.mdx

```diff
@@ -20,50 +20,72 @@ Cloudflare is part of [Winter CG](https://wintercg.org/), is more similar to the
+| `node:http`                | <AlertIcon /> | Provided as partially functional JS shims based on `fetch`                  |
+| `node:https`               | <AlertIcon /> | Provided as partially functional JS shims based on `fetch`                  |
+| `node:module`              | <AlertIcon /> | `SourceMap` is unimplemented, Implemented partially with JS stubs           |
+| `node:net`                 | <AlertIcon /> | `Server` and `BlockList` are unimplemented, sockets are partially supported |
+| `node:os`                  | <YesIcon />   | Provided as JS stubs                                                        |
+| `node:path`                | <YesIcon />   |                                                                             |
+| `node:path/posix`          | <YesIcon />   |                                                                             |
+| `node:path/win32`          | <YesIcon />   |                                                                             |
+| `node:process`             | <YesIcon />   | Provided as JS stubs                                                        |
+| `node:punycode`            | <NoIcon />    |                                                                             |
+| `node:querystring`         | <YesIcon />   |                                                                             |
+| `node:readline`            | <NoIcon />    | Provided as non-functional JS stubs                                         |
+| `node:readline/promises`   | <NoIcon />    | Provided as non-functional JS stubs                                         |
+| `node:stream`              | <YesIcon />   |                                                                             |
+| `node:stream/consumers`    | <YesIcon />   |                                                                             |
+| `node:stream/promises`     | <YesIcon />   |                                                                             |
+| `node:stream/web`          | <YesIcon />   |                                                                             |
+| `node:string_decoder`      | <YesIcon />   |                                                                             |
+| `node:timers`              | <YesIcon />   |                                                                             |
+| `node:timers/promises`     | <YesIcon />   |                                                                             |
+| `node:tls`                 | <NoIcon />    | Provides JS stubs but is unimplemented                                      |
+| `node:trace_events`        | <AlertIcon /> | Provided as non-functional JS stubs                                         |
+| `node:tty`                 | <YesIcon />   | Provided as JS shims redirecting output to the Console API                  |
```

Comment:
Should we briefly explain the difference between a "stub" and a "shim".

Reply:
Sure that's fair, and if anyone's still confused they can always google (or ask AI 🙈 ) what a JS shim is